src/OSPSuite.Presentation/Presenters/ParameterIdentifications/ParameterIdentificationIdentificationParametersPresenter.cs

```diff
@@ -114,10 +117,29 @@ public void UpdateStartValueFromSimulation(IdentificationParameterDTO identifica
 
       public void ChangeName(IdentificationParameterDTO identificationParameterDTO, string oldName, string newName)
       {
+         var matchedParameter = _parameterIdentification.AllIdentificationParameters.Single(x => x.Name == oldName);
```

Comment:
The `Single` makes me nervous. Do we know for certain there will be one with that name?